src/main/java/com/urlshortner/urlshortner/controller/CounterController.kt

```diff
@@ -14,9 +15,12 @@ class CounterController {
     @Autowired
     var counterService: CounterService? = null
```

Comment:
remove counter service, we will now use `RangeService` since it has zookeeper integrated.

Reply:
Let it be..since we will be using counterValue from counterService

Reply:
got it